lib/rules/web/admin_console/4.7/goto.xyaml

```diff
@@ -10,6 +10,11 @@ goto_resource_page_under_default_project:
 goto_project_resources_page:
   url: k8s/cluster/projects/<project_name>/workloads
   action: wait_box_loaded
+  action:
+    if_element:
+      selector:
+        xpath: //button[@data-test-id='topology-view-shortcuts']
+    ref: click_topology_view_switcher
```

Comment:
when a normal user visits project workloads page, by default topology view is shown and we need to switch to list view